cmd/run.go

```diff
@@ -201,12 +201,25 @@ func (cmd *RunCmd) RunRun(f factory.Factory, args []string) error {
 }
 
 func getCommands(f factory.Factory, logger log.Logger) ([]*latest.CommandConfig, error) {
+	// get current working dir
+	cwd, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+
 	// Set config root
 	configLoader := f.NewConfigLoader("")
-	configExists, err := configLoader.SetDevSpaceRoot(logger)
+	configExists, err := configLoader.SetDevSpaceRoot(log.Discard)
 	if err != nil {
 		return nil, err
 	}
+
+	// set working dir back to original
+	err = os.Chdir(cwd)
```

Comment:
please put this as an defer that's executed on function exit

Reply:
okay